war/src/main/less/modules/panes-and-bigtable.less

```
@@ -104,6 +104,11 @@ table.bigtable {
     text-align: right;
   }
 }
+.bigtable tfoot th,
+.bigtable .sortbottom th {
+  color: var(--text-color);
+  background-color: var(--white);
```

thoughts on just using the application background color?
```suggestion
  background-color: var(--background);
```

---

Sounds reasonable. Maybe even transparent would work